### ISSUE TITLE
Set maximum text width for accessibility/readability

### ISSUE THREAD

**Issue by jvine** [sul-dlss/purl#32]
Without a sidebar, text blocks can fill the whole width of the page - 200 characters or more - not optimal for accessibility/readability. Target should be 50 - 100 chars per line. 

`<dd>` is currently set to max-width: 45em, which is about 100 chars - same should be applied to `<p>` and `<li>`

(Mostly affects abstracts - any content that's not in a dl.)

**Comment by cbeer** [comment]
What about setting the `div.record-sections` to be `.col-md-8` (matching the searchworks column width)?

**Comment by jvine** [comment]
You could do that AND the 45em... :-)  It shouldn't be purl-specific. It's on my list for SW 3.1 and there's a partial ticket already (640). 8 cols is about 120 characters at the widest, so it's much better but still technically out of range.

**Comment by cbeer** [comment]
:+1:
